parsers/dataField/utils.go

```diff
@@ -0,0 +1,39 @@
+package datafield
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"math/big"
+)
+
+const (
+	esdtIdentifierSeparator  = "-"
+	esdtRandomSequenceLength = 6
```

Comment:
Perhaps esdtRandomIdentifierSequenceLength ?

Reply:
I will keep as it is